utils/mega_data_setup.py

```
@@ -438,6 +442,55 @@ def __set_desired_ilvl(
 
         return snipe_info, ilvl_info["ilvl"]
 
+    def __set_desired_pet_ilvl_list(self, path_to_data=None):
+        item_list_name = "desired_pet_ilvl_list"
+        file_name = f"{item_list_name}.json"
+        env_var_name = item_list_name.upper()
+        if path_to_data == None:
+            pet_ilvl_info = json.load(open(f"AzerothAuctionAssassinData/{file_name}"))
```

Ruff (0.8.0)

Use a context manager for opening files

(SIM115)